scrapers/PlayWithRae.yml

```diff
@@ -0,0 +1,31 @@
+name: PlayWithRae
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - playwithrae.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="videoDetails clear"]/h3
+      Date:
+        selector: //div[@class='videoInfo clear']/p[contains(span,"Date Added:")]/text()
+        postProcess:
+          - parseDate: January 2, 2006
+      Performers:
+        Name: //li[@class="update_models"]//a
+      Details:
+        selector: //div[@class="videoDetails clear"]/h3/following-sibling::text()
+      Tags:
+        Name: //li[@class="label" and contains(text(), "Tags:")]/following-sibling::text()
```

Comment:
Instead of `::text()`,`::li/a` could be used here
It works ok though so no biggie